docs/examples/ambassador-ingress.md

```diff
@@ -1,186 +1,181 @@
+    kind:  Mapping
+    metadata:
+      name: petstore
+      namespace: ambassador
     spec:
-      containers:
-        - name: petstore-backend
-          image: docker.io/swaggerapi/petstore3:unstable
-          ports:
-            - name: http
-              containerPort: 8080
----
-apiVersion: getambassador.io/v2
-kind:  Mapping
-metadata:
-  name: petstore
-  namespace: ambassador
-spec:
-  prefix: /petstore/
-  service: petstore
-```
-
-Once you've created this, apply it:
-
-```sh
-kubectl apply -f petstore.yaml
-service/petstore created
-deployment.apps/petstore created
-mapping.getambassador.io/petstore created
-```
-
-and you should be able to curl the service:
-
-```shell
-curl -k 'https://localhost/petstore/api/v3/pet/findByStatus?status=available'
-[{"id":1,"category":{"id":2,"name":"Cats"},"name":"Cat 1","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag1"},{"id":2,"name":"tag2"}],"status":"available"},{"id":2,"category":{"id":2,"name":"Cats"},"name":"Cat 2","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag2"},{"id":2,"name":"tag3"}],"status":"available"},{"id":4,"category":{"id":1,"name":"Dogs"},"name":"Dog 1","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag1"},{"id":2,"name":"tag2"}],"status":"available"},{"id":7,"category":{"id":4,"name":"Lions"},"name":"Lion 1","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag1"},{"id":2,"name":"tag2"}],"status":"available"},{"id":8,"category":{"id":4,"name":"Lions"},"name":"Lion 2","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag2"},{"id":2,"name":"tag3"}],"status":"available"},{"id":9,"category":{"id":4,"name":"Lions"},"name":"Lion 3","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag3"},{"id":2,"name":"tag4"}],"status":"available"},{"id":10,"category":{"id":3,"name":"Rabbits"},"name":"Rabbit 1","photoUrls":["url1","url2"],"tags":[{"id":1,"name":"tag3"},{"id":2,"name":"tag4"}],"status":"available"}]
-```
-
-or you can open https://localhost/petstore/ in your browser and change the URL of the specification to
-https://localhost/petstore/api/v3/openapi.json (since we mapped it to the /petstore prefix). 
-
-If you navigate to the Mappings part of the Ambassador Console (opened above) you will see the corresponding 
-PetStore mapping as configured.
-
-## Summary
-
-This should get you all set with running Ambassador under k0s. If you're not running under Docker just skip the docker-related
-steps above - but make sure that you have updated the k0s configuration in the same way as above. 
-
-If you're stuck with or have any questions about Ambassador please try the [Ambassador Slack](http://d6e.co/slack) to get help.
\ No newline at end of file
+      prefix: /petstore/
+      service: petstore
+    ```
+
+2. Apply the yaml file:
+
+    ```sh
+    kubectl apply -f petstore.yaml
+    service/petstore created
+    deployment.apps/petstore created
+    mapping.getambassador.io/petstore created
+    ```
+
+3. Either curl the service:
```

Comment:
This could be something like "validate the service is running",  continued with "in the terminal by using curl" and "or using a browser by opening https://....."